### ISSUE TITLE
bug: routerDirection="none" not setting page as root page

### ISSUE THREAD

**Issue by yasar-salim** [ionic-team/ionic#19982]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

When using routerDirection= "none" the page is not set as the root page. while pressing the back button it goes back to the previous page. I'm using ionic react sidemenu app.

**Expected behavior:**

We must be able to set the page as rootpage so that pressing back button wont go back to previous page.

**Steps to reproduce:**

install ionic react sidemenu app and check default navigation

**Related code:**

```
IonItem routerLink={appPage.url} routerDirection="none"
```

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI       : 5.4.6 (/usr/local/lib/node_modules/ionic)
   Ionic Framework : @ionic/react 4.11.5

Capacitor:

   Capacitor CLI   : 1.3.0
   @capacitor/core : 1.3.0

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v10.14.2 (/usr/local/bin/node)
   npm    : 6.13.1
   OS     : macOS Catalina
```

**Comment by yasar-salim** [comment]
@mlynch @mhartington  any resolutions?

**Comment by elylucas** [comment]
Setting routerDirection to none is meant to specify that there should be no animated transition to the new page, not to signify that its now the top of the routing stack.

Setting it to "root" isn't currently supported in Ionic/React, so we will look into what it will take to do so.

**Comment by elylucas** [comment]
fixed in 4.11.6
